client-js/app/elements/labrad-registry.html

```diff
@@ -8,15 +8,24 @@
     :host {
       display: block;
     }
-    #reg-menu {
+
+    .z-container {
+      background: white;
+      height: 100%;
+      margin: 0px;
+    }
+
+    .reg-menu {
       position: fixed;
-       @apply(--shadow-elevation-3dp);
-       padding-top: 5px;
-       padding-bottom: 5px;
-       background-color: white;
-       z-index: +1;
+      top: 140px;
+      left: 33px;
```

Comment:
this is unfortunate; I mean, it works for now, but magic numbers like this are a pain to tweak, e.g. if the toolbar changes  or fonts or any number of other things.